### PR TITLE
EDU-599-Markdown-inputs

### PR DESCRIPTION
**Pull request by ahelmberger**

### CODE REVIEW

File: src/components/markdown-help.less [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+  &.MarkdownHelp--small {
+    font-size: 12px;
+  }
+
+  &.MarkdownHelp--block {
+    padding: 4px 12px;
+    border-radius: 5px;
+    background-color: @edu-secondary-color-light;
+    opacity: 0.25;
```
Comment (adrianaluchian) [inline src/components/markdown-help.less:20]: I guess this opacity could be extracted in a semantically meaningful variable, something like `@edu-markdown-[inactive/unfocused]-opacity` 🤔

Comment (ahelmberger) [reply src/components/markdown-help.less]: Good idea, as it is used in two places. Done!

File: src/components/markdown-input.less [hunk 2]
```diff
@@ -0,0 +1,74 @@
+@markdown-input-block-min-height: 100px;
+
+.MarkdownInput {
+  display: grid;
+  gap: 5px;
+  grid-template-columns: 1fr;
```
Comment (adrianaluchian) [inline src/components/markdown-input.less:6]: We'd need to let this guy take `width: 100%` of its parent, for cases like this:

Screen Shot 2022-06-14 at 14 09 00
👇 
Screen Shot 2022-06-14 at 14 09 40

Comment (ahelmberger) [reply src/components/markdown-input.less:7]: True, thanks, now it takes 100%

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**